.husky/commit-msg.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+
+const commitMsgFile = process.argv[2];
+const commitMsg = fs.readFileSync(commitMsgFile, 'utf8');
+const testRegExp = /^\[MID-[0-9]+\] /g;
+
+const result = testRegExp.exec(commitMsg);
+if (result === null) {
+  console.log('Commit message should be of the form: "[MID-<ticket number>] <commit message text>"');
+  console.log('Example: "[MID-123] add this cool feature"');
+  console.log(`Provided commit message ${commitMsg}`);
+  process.exit(1);
+}
+process.exit(0);
```

Comment:
Consider including additional check - for max or min words/character in commit message. It should make commits smaller, smarter and consistent.

Reply:
I've asked our Tech Lead if we need this change. Currently we decided that it's unnecessary to add such restriction.